PowerUp/gradle.properties

```diff
@@ -15,4 +15,5 @@
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
\ No newline at end of file
+# org.gradle.parallel=true
+android.enableAapt2=false
```

Comment:
You might wanna add a line at the end.

Reply:
@m-murad I din't exactly get it , can you please explain ?

Reply:
there should always be a blank line at the end of the file.

Reply:
This is a convention most of the orgs follow and I saw it being followed in other systers projects.

Reply:
@m-murad I've added a blank line at end in gradle.properties file.